Update README.rst (#3188)
boto/boto3: README.rst

Getting Started
---------------

> Assuming that you have Python and ``virtualenv`` installed and that you set up your environment with:       

Reviewer: ```suggestion
Assuming that you have a supported version of Python installed, you can first set up your environment with:     

  ..code-block:: sh
      $ python -m venv .venv
      ...
      $ . .venv/bin/activate  
```

.. code-block:: sh
    
    $ virtualenv venv
    ...
    $ . venv/bin/activate

> you can install boto3 with:

Reviewer: ```suggestion
Then, you can install boto3 from PyPI with:
```

.. code-block:: sh

    $ python -m pip install boto3

> or with:

Reviewer: ```suggestion
or install from source with:
```

Author: Perfect! Thanks!
